docs/api-guide.md

```diff
@@ -241,6 +241,19 @@ You can get table metadata as follows:
 // Get the table metadata for "ns.tbl".
 TableMetadata tableMetadata = admin.getTableMetadata("ns", "tbl");
 ```
+### Repair a table
+
+You can repair table metadata of an existing table as follows:
```

Comment:
```suggestion
You can repair the table metadata of an existing table as follows:
```

Reply:
Thank you for your comments. I applied them all.